mongo/readpref/options.go

```diff
@@ -58,3 +58,12 @@ func WithTagSets(tagSets ...tag.Set) Option {
 		return nil
 	}
 }
+
+// WithHedgeEnabled specifies whether or not hedged reads should be enabled in the server.
+// TODO: we need something to link to here to explain what hedged reads are and what the server defaults are.
```

Comment:
Let's the docs team for a stable URL that we can use here.  Often they can provide one even if the page has not been created yet.

Reply:
Added a link to https://docs.mongodb.com/master/release-notes/4.4/#hedged-reads. That page also links to https://docs.mongodb.com/master/core/read-preference-hedge-option/#read-preference-hedged-read, which talks about the read preference option for hedging.